lib/message/compat.js

```diff
@@ -4,6 +4,11 @@ const protonmailCryptoTailMessage = '---END ENCRYPTED MESSAGE---';
 const protonmailCryptoHeaderRandomKey = '---BEGIN ENCRYPTED RANDOM KEY---';
 const protonmailCryptoTailRandomKey = '---END ENCRYPTED RANDOM KEY---';
 
+/**
+ * Extract armored encrypted message from email
+ * @param {String|Object} EmailPM
```

Comment:
I know the function kinda sort accepts it, but do we ever pass an object here? If not I don't really think we should start explicitly allowing it now. Maybe we can even get rid of the checks at some point
```suggestion
 * @param {String} EmailPM
```